frontend/src/app/data-entry/agent-form/agent-identification-form/agent-identification-form.component.ts

```diff
@@ -45,4 +91,48 @@ export class AgentIdentificationFormComponent implements OnChanges, OnDestroy {
             isGroup: data.agentDescription?.isGroup || false,
         });
     }
+
+    private isValid(): boolean {
+        return this.form.valid;
+    }
+
+    private toMutationInput([data, id]: [Partial<FormData>, string]): UpdateAgentInput {
```

Comment:
It's a non-issue here because there are not disabled controls, but in general, if you did disable controls, I would expect that those values are not specified in the update request.